api/python/provisioner/commands/setup_provisioner.py

```diff
@@ -651,7 +651,8 @@ def _prepare_local_repo(self, run_args, repo_dir: Path):
             repo_tgz_path,
             project_path=run_args.local_repo,
             version=run_args.prvsnr_version,
-            include_dirs=['pillar', 'srv', 'files', 'api', 'cli']
+            # include_dirs=['pillar', 'srv', 'files', 'api', 'cli']
+            include_dirs=['pillar', 'srv', 'files', 'cli']
```

Comment:
`pip3 install -U git+https://github.com/Seagate/cortx-prvsnr@cortx-1.0#subdirectory=api/python`
doesn't place files in `/opt/seagate/cortx/provisioner/api` anymore nor does python36-cortx-prvsnr.